lib/blockchain_test.go

```diff
@@ -451,7 +452,7 @@ func _getBalanceWithView(t *testing.T, chain *Blockchain, utxoView *UtxoView, pk
 
 func TestBalanceModelBlockTests(t *testing.T) {
 	setBalanceModelBlockHeights(t)
-
+	DeSoTestnetParams.ForkHeights.ProofOfStake2ConsensusCutoverBlockHeight = math.MaxUint32
```

Comment:
Does it make sense to move this block height update into `setBalanceModelBlockHeights` so it can be reverted in the cleanup? It seemed like part of the cause for the sporadic CI failures were the place where we set fork heights before a test, but never reverted them.

Reply:
the issue is that we started setting the `ProofOfStake2ConsensusCutoverBlockHeight` in `setBalanceModelBlockHeights` and this test assumes we're still using PoW blocks. I think it's cleaner to just manually set the fork height to a max uint32 here instead of creating a separate `setPoSBlockHeights` helper function. Lmk what you think. I can add a comment above that line saying 

```
This test assumes we're using PoW blocks and thus we need to set the PoS cutover fork height to some distant future height
```

Reply:
Sounds good to me